src/ploigos_step_runner/step_implementers/static_code_analysis/sonarqube.py

```diff
@@ -186,23 +188,35 @@ def _run_step(self):
         """
         step_result = StepResult.from_step_implementer(self)
 
-        # Optional: username and password
+        # Optional: token
```

Comment:
The original comment is still valid too; maybe include both? ("Optional: username and password" above the first two, "Optional: token" above the third)

Reply:
(reading further down, I see that comment still lives on line 199; might be worth either dragging that one up here, or pushing this comment down to 197)